LaTeX/LaTeX.sublime-syntax

```diff
@@ -225,6 +225,8 @@ contexts:
         - include: general-commands
         - include: macro-braces
 
+    - include: array-preamble
```

Comment:
This should be at the [packages context](https://github.com/Elliot2560/Packages/blob/65b6d484b4de3bc79d8150f582eeb1c46332968d/LaTeX/LaTeX.sublime-syntax#L772)